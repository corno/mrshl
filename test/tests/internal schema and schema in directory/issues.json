[
    [ "ignoring internal schema", "warning", 1, 1, 1, 1],
    [ "expected '(' but found '{'", "warning", 18, 1, 18, 2],
    [ "unexpected property: 'a'", "error", 19, 5, 19, 8],
    [ "expected 'number' but found 'string'", "error", 20, 10, 20, 13],
    [ "expected ')' but found '}'", "warning", 22, 1, 22, 2],
    [ "missing property: 'a1'", "error", 18, 1, 18, 2]
]