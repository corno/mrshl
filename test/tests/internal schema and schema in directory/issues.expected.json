[
    [ "schema retrieval", "warning", "found both external and internal schema. ignoring internal schema", null, null, null, null],
    [ "deserialization", "warning", "expected a verbose type: ( )", 18, 1, 18, 2],
    [ "deserialization", "error", "unknown property: 'a'. Choose from 'a26', 'b', 'c'", 19, 5, 19, 8]
]